Header header

uint8 OUT_OF_RANGE = 0
uint8 WITHIN_RANGE = 1

uint8 status
float32 estimated_range
float32 ground_truth_range
